packages/webapp/app/helloworld/client/client.tsx

```diff
@@ -6,29 +6,26 @@ import {
   useAppAuth,
   AppAuthActionsKeys,
 } from '@dailydotdev/shared/src/features/common/hooks/useAppAuth';
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
 
 export const ClientTest: FC = () => {
-  const { user, isLoggedIn, dispatch } = useAppAuth();
-
-  if (!isLoggedIn) {
-    return null;
-  }
+  const { user, dispatch } = useAppAuth();
 
   return (
-    <p>
+    <div>
```

Comment:
Should we add some gap in between the elements?

Reply:
I'll remove this component once intermediate pr will be ready, this is just for testing purposes 👀

Reply:
Ah, cool. No worries then.